mobile/src/components/course/components/edit-course/edit-course.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC } from 'react';
+
+import { AppScreenName, DataStatus } from '~/common/enums/enums';
+import { CourseUpdateCategoryRequestDto } from '~/common/types/types';
+import { BackButton, Spinner, View } from '~/components/common/common';
+import { EditCategory } from '~/components/course/components/components';
+import {
+  useAppDispatch,
+  useAppNavigate,
+  useAppSelector,
+  useCallback,
+  useEffect,
+  useFocusEffect,
+} from '~/hooks/hooks';
+import {
+  categoryActions,
+  coursesActions,
+  coursesManagementActions,
+} from '~/store/actions';
+
+import { styles } from './styles';
+
+const EditCourse: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const {
+    course,
+    dataStatus,
+    categories,
+    categoryDataStatus,
+    isFromCoursesManagement,
+  } = useAppSelector(({ courses, categories, coursesManagement }) => ({
+    course: courses.course,
+    dataStatus: courses.dataStatus,
+    categoryDataStatus: categories.dataStatus,
+    categories: categories.allCategories,
+    isFromCoursesManagement: coursesManagement.navigateFromCoursesManagement,
+  }));
+  const courseCategoryId = course?.courseCategoryId;
+
+  const toScreen = isFromCoursesManagement
+    ? AppScreenName.COURSE_MANAGEMENT
+    : AppScreenName.COURSE;
+
+  const goBack = (): void => {
+    navigation.navigate(toScreen);
```

Comment:
Why not to use navigation.goBack()?

Reply:
@Liza-Veis Then we will go to the courses screen

Reply:
then use navigation.pop()